tests/appsec/test_fingerprinting.py

```diff
@@ -82,6 +83,7 @@ def test_session(self):
 
 @rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.32nt1jz5tm2n")
 @features.fingerprinting
+@scenarios.remote_config_mocked_backend_asm_dd_nocache
```

Comment:
Required to enable the RC endpoint in the agent.

Reply:
with that, you'll run this test on the `remote_config_mocked_backend_asm_dd_nocache` scenario, but not on the default scenario, is it what you want to achieve ?

Reply:
The default scenario does not come with the RC endpoint enabled in the agent, that's the reason for using this scenario. Is there a better way to do it?